### PR TITLE
[HomePage] Add home page static list card

### PR DESCRIPTION
**Pull request by raintygao**
### Description

Add home page static list card

## Screenshot

image

## Testing the changes

Enable new homepage and go to homepage.

## Changelog

- feat: Add home page static list card

### Check List

- [ ] All tests pass
  - [ ] `yarn test:jest`
  - [ ] `yarn test:jest_integration`
- [ ] New functionality includes testing.
- [ ] New functionality has been documented.
- [ ] Update [CHANGELOG.md](./../CHANGELOG.md)
- [ ] Commits are signed per the DCO using --signoff

### CODE REVIEW

File: src/plugins/home/public/application/components/home_list_card.tsx [hunk 1]
```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import React from 'react';
+import { EuiCard, EuiLink, EuiListGroup } from '@elastic/eui';
+
+export const LEARN_OPENSEARCH_CONFIG = {
+  title: 'Learn Opensearch',
```
Comment (SuZhou-Joe) [inline src/plugins/home/public/application/components/home_list_card.tsx:10]: Shall we at least wrap it with i18n?

File: src/plugins/home/public/application/components/home_list_card.tsx [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+        display="plain"
+        footer={
+          config.allLink ? (
+            <EuiLink href={config.allLink} external={true} target="_blank">
+              view all
+            </EuiLink>
+          ) : null
+        }
+      >
+        <EuiListGroup listItems={config.list} color="text" size="s" />
```
Comment (ruanyl) [inline src/plugins/home/public/application/components/home_list_card.tsx:68]: If `description` is needed, I think we should use DescriptionList component https://oui.opensearch.org/1.8/#/display/description-list

**Review by ruanyl — APPROVED** [review]

**Comment by SuZhou-Joe** [comment]
Nit: Shall we move the View all to the bottom of the card?
image

**Review by SuZhou-Joe — APPROVED** [review]

**Comment by raintygao** [comment]
> Nit: Shall we move the View all to the bottom of the card? image

image

According to latest design, it shows that this link is not docked to bottom. So i didn't implement this for now.

**Comment by raintygao** [comment]
> > Nit: Shall we move the View all to the bottom of the card? image
> 
> image
> 
> According to latest design, it shows that this link is not docked to bottom. So i didn't implement this for now. But im glad to use some style like flex to make it docked to bottom in next update if needed.

Have added a comment to confirm with designers.

**Pull request merged**